Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Build a notary-server container without the build environment
+
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY ./notary-server/notary-server /bin/notary-server
+COPY ./notary-server/config.json /etc/docker/notary-server/config.json
+COPY ./notary-server/fixtures /fixtures
```

Comment:
nah thats why i understand this way :)

On Fri, Jul 31, 2015 at 9:37 AM, Aaron Lehmann <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/docker/notary-server-image/pull/1#discussion_r35990751>
> :
>
> > @@ -0,0 +1,15 @@
> > +# Build a notary-server container without the build environment
> > +
> > +FROM ubuntu:14.04
> > +
> > +RUN apt-get update && \
> > +    apt-get install -y ca-certificates && \
> > +    rm -rf /var/lib/apt/lists/*
> > +
> > +COPY ./notary-server/notary-server /bin/notary-server
> > +COPY ./notary-server/config.json /etc/docker/notary-server/config.json
> > +COPY ./notary-server/fixtures /fixtures
>
> Is there a way to avoid the layers without copying the files to the same
> place?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/notary-server-image/pull/1/files#r35990751>.
>